src/gbd_mapping_generator/cause_builder.py

```diff
@@ -30,10 +30,10 @@ def get_base_types():
                       ('incidence_consistent', "Union['bool', 'None']"),
                       ('deaths_consistent', "Union['bool', 'None']"),
                       ('birth_prevalence_consistent', "Union['bool', 'None']"),
-                      ('prevalence_aggregated', "Union['bool', 'None']"),
-                      ('incidence_aggregated', "Union['bool', 'None']"),
-                      ('deaths_aggregated', "Union['bool', 'None']"),
-                      ('birth_prevalence_aggregated', "Union['bool', 'None']"),
+                      ('prevalence_aggregates', "Union['bool', 'None']"),
```

Comment:
I copied this from somewhere else everywhere but started removing the quotes. I think these are supposed to go in to the template as is, not as string values, right ?